operator-common/src/main/java/io/strimzi/operator/common/AbstractOperator.java

```diff
@@ -59,10 +64,46 @@
     protected final S resourceOperator;
     private final String kind;
 
+    private final MeterRegistry metrics;
+    private final Counter periodicReconciliationsCounter;
+    private final Counter reconciliationsCounter;
+    private final Counter failedReconciliationsCounter;
+    private final Counter successfulReconciliationsCounter;
+    private final AtomicInteger resourceCounter;
+
     public AbstractOperator(Vertx vertx, String kind, S resourceOperator) {
         this.vertx = vertx;
         this.kind = kind;
         this.resourceOperator = resourceOperator;
+
+        // Setup metrics
+        metrics = BackendRegistries.getDefaultNow();
+        
+        periodicReconciliationsCounter = Counter.builder("strimzi.periodical.reconcilitions")
```

Comment:
if we could assume that `strimzi.reconciliations` is a sort of root for the reconciliations related metrics, should the others be `strimzi.reconciliations.periodical` and not the other way around?